.gitignore

```diff
@@ -34,3 +34,6 @@
 /yarn-error.log
 yarn-debug.log*
 .yarn-integrity
+
+# Ignore spec helpers
+spec/helpers/*
```

Comment:
Out of curiosity, why are we ignoring the `spec/helpers` folder? I'm not saying it's wrong, I'm just interested in the rationale.

Reply:
I'm not using the helper files. Because they are automatically created and because of the comments that are in them by default, they kept being added to my PRs as blank files.

Figured I'd cut it out at the source ¯\_(ツ)_/¯

Reply:
Fair enough. If they're autogenerated and you're not using them that's fine for now. Though we may need to revisit this in the future if we decide to use them at some stage.